### DISCUSSION TITLE
PocketBase + HTMX

### DISCUSSION THREAD

**Discussion by tinsever** [pocketbase/pocketbase#5348] [Q&A] [answered]
Hello everyone,
I am having the idea of using PocketBase with HTMX. but I'm sorry, I don't wanna use GoLang at all. 
Is there any "smart-ish" way to do that?
Thanks!

**Comment by tinsever** [comment]
To make this question rather simple: **Can I use auth etc. with server-side pocketbase only without much more code?**

**Reply by duncsully** [reply, marked as answer]
There's a whole section on using [JS to extend Pocketbase, e.g. adding custom routes](https://pocketbase.io/docs/js-routing/) that'll probably suffice for your purposes.

**Comment by ccutch** [comment]
I tried this before and it works well using the go extensions to serve templates (which i like to embed in the executable). Would recommend and checkout ccutch/pbj

**Reply by tinsever** [reply]
Don't wanna learn go, yet.
